devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -238,7 +332,32 @@ def zwaveEvent(physicalgraph.zwave.commands.batteryv1.BatteryReport cmd) {
 
 def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd) {
 	def isActive = false
-	if (cmd.notificationType == 0x0E) { //Siren notification
+	def result = []
+
+	if (cmd.notificationType == 0x07) { // Burglar notification
+		def map = [name: "tamper", value: "detected"]
+		map.data = [sirenName: deviceName]
+		switch (cmd.event) {
+			case 0:
+				map.value = "clear"
```

Comment:
Forgot that tamper clear doesn't work when the device is locally executing.

See Tom's comment here:
https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/3619

Reply:
You'll have to automatically clear it. It's not ideal, but it's the best workaround we have atm.